compiler/include/codegen.h

```diff
@@ -92,6 +93,7 @@ struct GenInfo {
   // Once we get to code generation....
   llvm::Module *module;
   llvm::IRBuilder<> *builder;
+  llvm::MDBuilder *mdHelper;
```

Comment:
I wonder if we should name it mdBuilder? And consider renaming 'builder' to irBuilder?